backend/src/User.ts

```diff
@@ -5,6 +5,7 @@ import debug from 'debug'
 import Logger from './Logger'
 import EventBus from './EventBus'
 import path from 'path'
+import axios from 'axios'
 import { r3 } from './remote.it'
```

Comment:
```suggestion
```

Please completely remove the r3 dependency and module from the backend codebase and package.json. You will have to move over or re-write the logic in the backend user class.

Reply:
done